README.md

```diff
@@ -45,3 +45,6 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 ```
+## Add your repo to enacit packages
```

Comment:
Since enacit-ansible is private and this doc public, I'd say _Ask enacit-linux-sysadmins to give your repo read access to the deployment package_